p2pd/main.go

```diff
@@ -13,12 +13,13 @@ import (
 	ps "github.com/libp2p/go-libp2p-pubsub"
 	quic "github.com/libp2p/go-libp2p-quic-transport"
 	identify "github.com/libp2p/go-libp2p/p2p/protocol/identify"
+	multiaddr "github.com/multiformats/go-multiaddr"
 )
 
 func main() {
 	identify.ClientVersion = "p2pd/0.1"
 
-	sock := flag.String("sock", "/tmp/p2pd.sock", "daemon control socket path")
+	maddrString := flag.String("maddr", "/unix/tmp/p2pd.sock", "daemon control multiaddr")
```

Comment:
I want a better name for this argument; call it `-listen` perhaps?

Reply:
Changed to `listen`